src/content/operations/usage/_index.en.md

```diff
@@ -0,0 +1,825 @@
+
+##### 3. Consume the Service on **cluster2**
+
+Verify that the exported `nginx` Service was imported to **cluster2** as expected. Submariner (via Lighthouse) automatically creates a
+corresponding `ServiceImport`:
+
+```bash
+$ kubectl config use-context cluster2
+Switched to context "cluster2".
+```
+
+```bash
+$ kubectl get -n submariner-operator serviceimport
+NAME                        TYPE           IP                AGE
+nginx-nginx-test-cluster3   ClusterSetIP   [100.3.220.176]   13m
+```
+
+Next, run a test Pod on **cluster2** and try to access the `nginx` Service from within the Pod:
+
+```bash
+$ kubectl create namespace nginx-test
+namespace/nginx-test created
+
+$ kubectl -n nginx-test  run --generator=run-pod/v1 \
+tmp-shell --rm -i --tty --image quay.io/submariner/nettest -- /bin/bash
+```
+
+```bash
+bash-5.0# curl nginx.nginx-test.svc.clusterset.local:8080
+<!DOCTYPE html>
+<html>
+<head>
+<title>Welcome to nginx!</title>
+<style>
+    body {
+        width: 35em;
+        margin: 0 auto;
+        font-family: Tahoma, Verdana, Arial, sans-serif;
+    }
+</style>
+</head>
+<body>
+<h1>Welcome to nginx!</h1>
+<p>If you see this page, the nginx web server is successfully installed and
+working. Further configuration is required.</p>
+
+<p>For online documentation and support please refer to
+<a href="http://nginx.org/">nginx.org</a>.<br/>
+Commercial support is available at
+<a href="http://nginx.com/">nginx.com</a>.</p>
+
+<p><em>Thank you for using nginx.</em></p>
+</body>
+</html>
+```
+<!-- markdownlint-disable no-hard-tabs -->
+```bash
+bash-5.0# dig nginx.nginx-test.svc.clusterset.local
+; <<>> DiG 9.16.6 <<>> nginx.nginx-test.svc.clusterset.local
+;; global options: +cmd
+;; Got answer:
+;; WARNING: .local is reserved for Multicast DNS
+;; You are currently testing what happens when an mDNS query is leaked to DNS
+;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 34800
+;; flags: qr aa rd; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1
+;; WARNING: recursion requested but not available
+
+;; OPT PSEUDOSECTION:
+; EDNS: version: 0, flags:; udp: 4096
+; COOKIE: 6ff7ea72c14ce2d4 (echoed)
+;; QUESTION SECTION:
+;nginx.nginx-test.svc.clusterset.local. IN	A
+
+;; ANSWER SECTION:
+nginx.nginx-test.svc.clusterset.local. 5 IN A	100.3.220.176
+
+;; Query time: 16 msec
+;; SERVER: 100.2.0.10#53(100.2.0.10)
+;; WHEN: Mon Nov 30 17:52:55 UTC 2020
+;; MSG SIZE  rcvd: 125
+```
+<!-- markdownlint-enable no-hard-tabs -->
+Note that DNS resolution works across the clusters, and that the IP address **100.3.220.176** returned is the same ClusterIP associated with
+the `nginx` Service on **cluster3**.
+
+##### 4. Create an `nginx` Deployment on **cluster2**
+
+If multiple clusters export a Service with the same name and from the same namespace, it will be recognized as a single combined Service.
+To test this, let's deploy the same `nginx` Service, this time on the nginx-test namespace on **cluster2**:
```

Comment:
```suggestion
If multiple clusters export a Service with the same name and from the same namespace, it will be recognized as a single logical Service.
To test this, we'll deploy the same `nginx` Service in the same namespace on **cluster2**:
```